### PR TITLE
Disallow masking for background.

### LINKED ISSUE #796: Background doesn't complete fill up the canvas
## Bug Description

There's a thin line of pixels when setting an image as background, see screenshot

## Screenshots

Screen Shot 2020-03-26 at 4 32 17 AM

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
Fixes #796 

Background media should not be masked, it should always cover the full area.

If the user wants to add a "background shape" for some reason, Fill feature can be used.

### CODE REVIEW

File: assets/src/edit-story/masks/output.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export default function WithMask({
         ...style,
         clipPath: `url(#${maskId})`,
         // stylelint-disable-next-line property-no-vendor-prefix
-        WebkitClipPath: `url(#${maskId})`,
+        webkitClipPath: `url(#${maskId})`,
```
Comment (swissspidy) [inline assets/src/edit-story/masks/output.js]: Why was this changed? See #791

Comment (miina) [reply assets/src/edit-story/masks/output.js]: Eh? 😕  I didn't change this knowingly. Odd, for some reason locally it didn't update it correctly when merging. Will change manually.

Comment (miina) [reply assets/src/edit-story/masks/output.js]: Looks like it was changed automatically locally. Odd.

**Review by spacedmonkey — APPROVED** [review]
Works when testing locally.

File: assets/src/edit-story/masks/output.js (hunk 1 above)
Comment (barklund) [reply assets/src/edit-story/masks/output.js]: `npm run lint:css:fix assets/src/edit-story/masks/output.js` changes this file for me locally. Every. Time. FYI.

Comment (swissspidy) [reply assets/src/edit-story/masks/output.js]: Good to know, thanks! Will investigate!

**Review by wassgha — APPROVED** [review]

File: assets/src/edit-story/masks/output.js (hunk 1 above)
Comment (wassgha) [reply assets/src/edit-story/masks/output.js]: Tracking in https://github.com/google/web-stories-wp/issues/845 so that I can merge this now

**Comment by swissspidy** [comment]
Added to alpha branch

**Pull request merged**